run.py

```diff
@@ -204,13 +209,19 @@ def create_nodes(
             ceph_vmnodes = create_ibmc_ceph_nodes(
                 cluster, inventory, osp_cred, run_id, instances_name
             )
+        elif cloud_type == "baremetal":
+            ceph_vmnodes = create_baremetal_ceph_nodes(cluster, inventory)
 
         ceph_nodes = []
         for node in ceph_vmnodes.values():
             if cloud_type == "openstack":
                 private_key_path = ""
                 private_ip = node.get_private_ip()
                 look_for_key = False
+            elif cloud_type == "baremetal":
+                private_key_path = ""
```

Comment:
can we read this from the `Node`? This could be different from environment to environment.